app/assets/javascripts/components/revisions/diff_viewer.jsx

```diff
@@ -346,7 +346,7 @@ const DiffViewer = createReactClass({
           <div>
             <div className="diff-viewer-scrollbox">
               <strong>{salesforceButtons}</strong>
-              <table>
+              <table style={{ width: '100%' }}>
```

Comment:
I'd prefer this be done via the stylesheet, rather than setting the style at the component level.

Reply:
Ah good point. Fixed!